File: INTR A ARREGLOS.cpp
#include <iostream>
using namespace std;
int main() {
    // Declaración (tamaño fijo)
    int arreglo[5];  // valores indefinidos


    // Inicialización
    int arreglo2[5] = {10, 20, 30, 40, 50};


    // Asignar valor
    arreglo[0] = 99;


    // Recuperar valor
    int valor = arreglo2[2]; // 30


    // Recorrer arreglo
    for(int i = 0; i < 5; i++) {
        cout << "Índice " << i << " -> " << arreglo2[i] << endl;
    }


    return 0;
}

File: INTR A ARREGLOS.py
arreglo = []            # Lista vacía
arreglo = [10, 20, 30]  # Lista con valores iniciales


# Asignar un valor mediante índice
arreglo[0] = 99  # ahora el primer elemento vale 99


valor = arreglo[1]  # guarda 20


# Inicializar con valores por defecto
arreglo = [0] * 5   # crea [0, 0, 0, 0, 0]


# Leer un valor (ejemplo en bucle)
for i in range(len(arreglo)):
    print(f"Índice {i} -> {arreglo[i]}")
